### PR TITLE
Release version increased in pom.xml

### PR DESCRIPTION
**Pull request by shudhansu-shekhar**
### Description of the Change

Release version increased in pom.xml

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: <shudhansu shekhar> <shudhansu.shekhar.ext@ericsson.com>

### CODE REVIEW

File: wiki/markdown/usage/service.md [hunk 1]
```diff
@@ -250,6 +253,9 @@ It provides single endpoint for both REMReM Generate and REMReM Publish.
 The service works on the relative link `/generateAndPublish` if run as standalone
 application or `/publish/generateAndPublish` if run as Tomcat app.
 
+NOTE: Here, introduction of new REMReM property for the size of number of events in the array passed to /generateAndPublish endpoint.
```
Comment (z-sztrom) [inline wiki/markdown/usage/service.md:256]: Add property name.

File: wiki/markdown/usage/service.md [hunk 2]
```diff
@@ -250,6 +253,9 @@ It provides single endpoint for both REMReM Generate and REMReM Publish.
...
+Here, Maximum size of events in array is fetched from REMReM property and checked during publishing.
```
Comment (z-sztrom) [inline wiki/markdown/usage/service.md:257]: Remove.

**Review by z-sztrom — CHANGES_REQUESTED** [review]

File: wiki/markdown/usage/service.md [hunk 3]
```diff
@@ -214,7 +214,10 @@ GET
 ### `/producer/msg`
 This endpoint is used to publish already generated Eiffel event to message bus.
 
-Note: This endpoint will not validate the message. It will check only if the message contains eventId and eventType.
+Note: 
+1. This endpoint will not validate the message. It will check only if the message contains eventId and eventType.
+2. Here, introduction of new REMReM property for the size of number of events in the array passed to /producer/msg endpoint.
+   Here, Maximum size of events in array is fetched from REMReM property and checked during publishing.
```
[the remark this answers is not in the document]
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:220]: Done

File: wiki/markdown/usage/service.md (hunk 2 above)
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:257]: Done

File: wiki/markdown/usage/service.md (hunk 1 above)
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:256]: Done

File: wiki/markdown/usage/service.md [hunk 4]
```diff
@@ -214,7 +214,10 @@ GET
 ### `/producer/msg`
 This endpoint is used to publish already generated Eiffel event to message bus.
 
-Note: This endpoint will not validate the message. It will check only if the message contains eventId and eventType.
+Note: 
+1. This endpoint will not validate the message. It will check only if the message contains eventId and eventType.
+2. Here, introduction of new REMReM property for the size of number of events in the array passed to /producer/msg endpoint.
```
[the remark this answers is not in the document]
Comment (shudhansu-shekhar) [reply wiki/markdown/usage/service.md:219]: Done

**Review by shudhansu-shekhar — COMMENTED** [review]
Done, please check the latest code

**Review by z-sztrom — APPROVED** [review]

**Pull request merged**